usaspending_api/awards/management/commands/loadcontracts.py

```diff
@@ -26,25 +27,227 @@ def handle(self, *args, **options):
         field_map = {
             "federal_action_obligation": "dollarsobligated",
             "description": "descriptionofcontractrequirement",
+            "other_statutory_authority": "otherstatutoryauthority",
             "modification_number": "modnumber"
         }
 
         value_map = {
             "data_source": "USA",
-            "award": lambda row: Award.objects.get_or_create(piid=row['piid'], type='C')[0],
-            "recipient": lambda row: LegalEntity.objects.get_or_create(recipient_name=row['dunsnumber'])[0],
-            "awarding_agency": lambda row: Agency.objects.get(subtier_code=self.get_agency_code(row['maj_agency_cat'])),
+            "award": lambda row: self.create_or_get_award(row),
+            "awarding_agency": lambda row: self.get_agency(row),
             "action_date": lambda row: self.convert_date(row['signeddate']),
             "last_modified_date": lambda row: self.convert_date(row['last_modified_date']),
             "gfe_gfp": lambda row: row['gfe_gfp'].split(":")[0],
+            "multiple_or_single_award_idv": lambda row: row['multipleorsingleawardidc'].split(' ')[0].split(':')[0],
```

Comment:
I was having to change them around a lot. Also, due to the way the function passing works, you'd end up having to do something like 

`multiple_or_single_award_idv`: lambda row: self.parse_with_splits("multipleorsingleawardidv")`

Which is probably still slightly more readable, so this is kind of just a relic of hammering my head against it for a while.
Let's put it down for refactoring when I make the changes for DS-542?